addons/mail/i18n/mail.pot

```diff
@@ -7421,6 +7427,11 @@ msgstr ""
 msgid "Wrong operation name (%s)"
 msgstr ""
 
+#. module: mail
+#: model_terms:ir.ui.view,arch_db:mail.view_mail_form
+msgid "YYYY-MM-DD HH:MM:SS"
```

Comment:
I don't think we should let people translate that ^^

Reply:
I don't get what you mean by "they could think this is editable" for me it could be translated in other language (e.g. in french AAAA-MM-JJ HH:MM:SS)